src/track/beatutils.cpp

```diff
@@ -231,19 +242,31 @@ double BeatUtils::makeConstBpm(
                 ((kMaxSecsPhaseError * sampleRate) / numberOfBeats);
         const double maxRoundSamples = constantRegions[i].beatLength +
                 ((kMaxSecsPhaseError * sampleRate) / numberOfBeats);
-        if (longestRegionLength > minRoundSamples &&
-                longestRegionLength < maxRoundSamples) {
+        if (longestRegionBeatLenth > minRoundSamples &&
```

Comment:
Typo

Reply:
If you rename the variable anyway, it would be good if you'd append the unit (Samples?) to the variable name.

Reply:
All variables are on a frame base. 
I have added an initial comment to clarify this. And took this momentum to unify the variable naming inside the function.